modules/Movecraft/src/main/java/net/countercraft/movecraft/async/rotation/RotationTask.java

```diff
@@ -458,6 +461,22 @@ private boolean checkChests(Material mBlock, MovecraftLocation newLoc) {
         return !testMaterial.equals(mBlock) || oldHitBox.contains(aroundNewLoc);
     }
 
+    private boolean withinWorldBorder(MovecraftLocation location) {
```

Comment:
Why can we not just convert the `MovecraftLocation` to a spigot `Location` and simply make use of `WorldBorder::isInside`?

Reply:
IsInside does not exist in 1.10.2 spigot @cccm5